packages/aws-cdk-lib/aws-ecs/test/aws-log-driver.test.ts

```diff
@@ -146,6 +148,20 @@ describe('aws log driver', () => {
 
   });
 
+  test('throws error when specifying maxBufferSize and blocking mode', () => {
```

Comment:
Can you add one more unit test to validate that this will throw when maxBufferSize is used and no blocking mode is specified that way we're validating that we can't specify maxBufferSize with the default driver mode